Tests/tests/test_integration_test.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from Tests.test_integration import __print_investigation_error
+from Tests.test_content import ParallelPrintsManager
+import demisto_client
+
+
+@pytest.mark.parametrize('command, output', [
+    ('!create account name="User" mail="Test" password="123123!"',
+     '  Command: !create account name="User" mail="Test" password=******'),
+    ('!create account name="User" mail="Test" Password="123123!"',
+     '  Command: !create account name="User" mail="Test" password=******'),
+    ('!create account name="User" mail="Test" pass word="123123!"',
+     '  Command: !create account name="User" mail="Test" pass word="123123!"'),
+])
+def test_print_investigation_error(command, output, mocker):
+    """
+    Given
+    - Failed command line
+    When
+    - When a playbook test fails
+    Then
+    - Ensure that it replaces the password="123123!" with password=******
+    - Ensure that it replaces the Password="123123!" with password=******
+    - Ensure that it  not replaces the "pass word="123123!""
```

Comment:
```suggestion
    - Ensure message sent to print manager contains `password=******` instead of `password="123123!"`
    - Ensure message sent to print manager contains `password=******` instead of `Password="123123!"`
    - Ensure message sent to print manager contains "pass word="123123!""
```